contracts/LosslessControllerV3.sol

```diff
@@ -67,38 +56,29 @@ contract LosslessControllerV3 is Initializable, ContextUpgradeable, PausableUpgr
         mapping(address => LocksQueue) queue;
     }
 
+    mapping(ILERC20 => TokenLockedFunds) private tokenScopedLockedFunds;
+    
     struct ReceiveCheckpoint {
         uint256 amount;
         uint256 timestamp;
     }
+    
+    uint256 private constant BY_HUNDRED = 1e2;
```

Comment:
Rename to hundred and maybe make public?

Reply:
Fixed on new commit 4bec697